Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -104,6 +104,16 @@ protected static IMixedRealityInputSystem InputSystem
         protected bool forceUpdate = false;
 
         // basic button states
+
+        //TODO: Enable PassThroughMode to enable events to continue up the hierarchy
```

Comment:
could you please share the whiteboard photos we took together when we discussed architecture? I don't recall where we landed, but not sure if passing through only interactables is what we wanted. I thought we wanted to have something more general.

Reply:
We built a scrolling object collection with our latest partner and were able to circumvent the "pass though mode" idea. In the end we didn't have to make any changes to the MRTK input layer.